yaml_specs/modulemd_packager_v3.yaml

```
@@ -0,0 +1,486 @@
+                # By default the build system only allows components listed under
+                # data.components.rpms to be built as part of this module.
+                # In case the expected RPM build names do not match the component
+                # names, the list can be defined here.
+                # This list overrides rather then just extends the default.
+                # List of package build names without versions.
+                #
+                # Type: OPTIONAL
+                whitelist:
+                    - fooscl-1-bar
+                    - fooscl-1-baz
+                    - xxx
+                    - xyz
+
+        # Alternate example with no dependencies
+        - context: CTX2
+          platform: f33
+
+    # references:
+    # References to external resources, typically upstream
+    #
+    # Type: OPTIONAL
+    references:
+        # community:
+        # Upstream community website, if it exists
+        #
+        # Type: OPTIONAL
+        community: http://www.example.com/
+
+        # documentation:
+        # Upstream documentation, if it exists
+        #
+        # Type: OPTIONAL
+        documentation: http://www.example.com/
+
+        # tracker:
+        # Upstream bug tracker, if it exists
+        #
+        # Type: OPTIONAL
+        tracker: http://www.example.com/
+
+    # profiles:
+    # Profiles define the end user's use cases for the module. They consist of
+    # package lists of components to be installed by default if the module is
+    # enabled. The keys are the profile names and contain package lists by
+    # component type. There are several profiles defined below. Suggested
+    # behavior for package managers is to just enable repository for selected
+    # module. Then users are able to install packages on their own. If they
+    # select a specific profile, the package manager should install all
+    # packages of that profile.
+    # Defaults to no profile definitions.
+    #
+    # Type: OPTIONAL
+    profiles:
+
+        # An example profile that defines a set of packages which are meant to
+        # be installed inside a container image artifact.
+        #
+        # Type: OPTIONAL
+        container:
+            rpms:
+                - bar
+                - bar-devel
+
+        # An example profile that delivers a minimal set of packages to
+        # provide this module's basic functionality. This is meant to be used
+        # on target systems where size of the distribution is a real concern.
+        #
+        # Type: Optional
+        minimal:
+            # A verbose description of the module, optional
+            description: Minimal profile installing only the bar package.
+            rpms:
+                - bar
+            # default (boolean):
+            # Whether this profile should be installed if none is explicitly
+            # specified. No more than one configuration may have `default: true`.
```

```suggestion
            # specified.
```

Apparently this restriction on the `default` property shouldn't be there, since any number of profiles can be installed by default.